chess-py-2.0/src/Board_class.py

```diff
@@ -40,8 +63,18 @@ def space_points_ref(self):
                 print_counter = 0
 
 
-chessboard = Board()
-chessboard.space_points_ref()
+    def board_init(self):
+        bR1 = Rook(Piece(True, 'rook', Position(0,0)))
+        bR2 = Rook(Piece(True, 'rook', Position(7,0)))
+        wR1 = Rook(Piece(False, 'rook', Position(0,7)))
+        wR2 = Rook(Piece(False, 'rook', Position(7,7)))
```

Comment:
I think I saw that in your Piece `__init__` you have the white/black value hard coded to `True`, so the first value you pass in doesn't make a difference